src/main/java/website/frontrow/board/Player.java

```diff
@@ -83,4 +88,50 @@ public void addToLog(String action)
     {
         Log.add(action);
     }
+    
+    /**
+     * A getter for lives.
+     * @return lives integer
+     */
+    public int getLives() 
+    {
+    	return lives;
+    }
+    
+    /**
+     * A setter for lives.
+     * @param l integer
+     */
+    public void setLives(int l)
+    {
+    	lives = l;
+    	addToLog("The player's current amount of lives is now: " + lives);
+    }
+    
+    /**
+     * Decreases the lives by one.
+     */
+    public void loseLife()
+    {
+    	lives--;
+    	addToLog("The player lost a life, total lives is now: " + lives);
+    }
+    
+    /**
+     * Increases the lives by one.
+     */
+    public void addLife()
+    {
+    	lives++;
+    	addToLog("The player earned a life, total lives is now: " + lives);
+    }
+    
+    /**
+     * Returns whether or not the player is alive.
+     * @return boolean
+     */
+    public boolean playerAlive()
```

Comment:
You should override the method isAlive() in Unit instead of creating a new method.

<interface, review